### ISSUE TITLE
Consider enabling DirectPlay via the installer

### ISSUE THREAD

**Issue by BC46** [oliverpechey/Freelancer-HD-Edition-Installer#136]
DirectPlay can be automatically enabled via the following command:
```
Dism /Online /Enable-Feature /FeatureName:DirectPlay /All /Quiet /NoRestart
```

**Comment by boltedev** [comment]
Yes, it can indeed. However, I am unsure why its necessary. Whenever I have tried to run Freelancer without it enabled, and indeed was able to reproduce this right now by temp disabling directplay - I get this popup.

![DirectPlay](https://github.com/oliverpechey/Freelancer-HD-Edition-Installer/assets/159622994/6715604c-2844-4a3b-9a92-2cb9e178be12)

I am however unsure if this popup occurs on Win10 or older however

**Comment by BolteDev** [comment]
To add to this, could use
`dism /online /get-featureinfo /featurename:DirectPlay` which will return info, amomg which is State: Enabled/Disabled. Then can run the aforementioned to enable it.

**Comment by BC46** [comment]
There was one user in the Discord who didn't get the popup when starting the game on Windows 10 Education. They were confused as to why the game wouldn't start because of this. Hence the suggestion to automatically enable DP in the installer.
